src/gui/transferlistfilters/trackersfilterwidget.cpp

```diff
@@ -175,12 +182,15 @@ void TrackersFilterWidget::refreshTrackers(const BitTorrent::Torrent *torrent)
     }
 
     item(ERROR_ROW)->setText(tr("Error (%1)").arg(m_errors.size()));
+    item(TRACKERERROR_ROW)->setText(tr("Tracker error (%1)").arg(m_trackerErrors.size()));
     item(WARNING_ROW)->setText(tr("Warning (%1)").arg(m_warnings.size()));
 
-    if (currentRow() == ERROR_ROW)
-        applyFilter(ERROR_ROW);
-    else if (currentRow() == WARNING_ROW)
-        applyFilter(WARNING_ROW);
+    if ((currentRow() == ERROR_ROW)
+            || (currentRow() == TRACKERERROR_ROW)
+            || (currentRow() == WARNING_ROW))
+    {
+        applyFilter(currentRow());
+    }
```

Comment:
To avoid repetitive function calls:
```suggestion
    if (const int row = currentRow(); (row == ERROR_ROW)
            || (row == TRACKERERROR_ROW)
            || (row == WARNING_ROW))
    {
        applyFilter(row);
    }
```